### ISSUE TITLE
Description of Achilles on the wiki-page

### ISSUE THREAD

**Issue by PRijnbeek** [OHDSI/AchillesWeb#60]
Hi All,

If i look at the [wiki ](page (http://www.ohdsi.org/web/wiki/doku.php?id=documentation:software:achilles) it says that Achilles is showing prevalences of conditions. I disagree, it is showing the frequency of condition codes. You cannot say that both are the same thing in my view.

I think it is very important we communicate this in the right way everywhere including the AchillesWeb interface because this very easily mis-interpreted.

Any thoughts?

Peter

**Comment by schuemie** [comment]
Hi @PRijnbeek,

Completely agree! @pbr6cornell, we had discussed this recently and had come up with a new label, but I forgot what it was. Do you remember?

Cheers,
Martijn

**Comment by pbr6cornell** [comment]
I agree as well, and recommend we revise text to read 'concept frequency'
